examples/python/hellostreamingworld/async_greeter_server.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Python AsyncIO implementation of the GRPC hellostreamingworld.MultiGreeter server."""
+
+import asyncio
+import logging
+
+import grpc
+from hellostreamingworld_pb2_grpc import MultiGreeterServicer, add_MultiGreeterServicer_to_server
+from hellostreamingworld_pb2 import HelloRequest, HelloReply
+
+
+async def say_hello_generator(request):
+    for i in range(10):
```

Comment:
We can make the magic number `10` a constant.

Reply:
done